src/core/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,817 @@
+  const { container } = render(<Slider values={defaultSingleValue} />);
+  assertBaseElement(container);
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.getAttribute('aria-disabled')).toEqual('false');
+});
+
+it('should reset callbacks on value change', () => {
+  const wrapper = render(<Slider values={defaultSingleValue} />);
+  const { container } = wrapper;
+  assertBaseElement(container);
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  wrapper.rerender(<Slider values={[10, 20]} max={90} />);
+});
+
+it('should not render thumbs if no values are defined', () => {
+  const { container } = render(<Slider values={[]} />);
+  assertBaseElement(container);
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeFalsy();
+});
+
+it('should render disabled component', () => {
+  const { container } = render(<Slider values={defaultSingleValue} disabled />);
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container.iui-disabled'),
+  ).toBeTruthy();
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.getAttribute('aria-disabled')).toEqual('true');
+});
+
+it('should render min max labels by default', () => {
+  const { container } = render(<Slider values={defaultSingleValue} />);
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+  expect(
+    (container.querySelector('.iui-slider-min') as HTMLSpanElement).textContent,
+  ).toBe('0');
+  expect(
+    (container.querySelector('.iui-slider-max') as HTMLSpanElement).textContent,
+  ).toBe('100');
+});
+
+it('should render specified min max labels', () => {
+  const { container } = render(
+    <Slider values={defaultSingleValue} min={5} max={55} />,
+  );
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+  expect(
+    (container.querySelector('.iui-slider-min') as HTMLSpanElement).textContent,
+  ).toBe('5');
+  expect(
+    (container.querySelector('.iui-slider-max') as HTMLSpanElement).textContent,
+  ).toBe('55');
+});
+
+it('should render provided min max labels', () => {
+  const { container } = render(
+    <Slider values={defaultSingleValue} maxLabel='big' minLabel='small' />,
+  );
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+  expect(
+    (container.querySelector('.iui-slider-min') as HTMLSpanElement).textContent,
+  ).toBe('small');
+  expect(
+    (container.querySelector('.iui-slider-max') as HTMLSpanElement).textContent,
+  ).toBe('big');
+});
+
+it('should render provided min max label nodes', () => {
```

Comment:
This test can be combined with the previous one, it's just checking ReactNode behavior

Reply:
merged 2 tests to check string and jsx